sunraster/spectrogram_sequence.py

```diff
@@ -1,13 +1,13 @@
-import textwrap
 import numbers
+import textwrap
 
 import numpy as np
-import astropy.units as u
-import ndcube.utils.sequence
 from ndcube import NDCubeSequence
 
```

Comment:
```suggestion
```

Reply:
If you want this, the isort config needs changing.

Reply:
Ah!  Is that what isort is to do with!  I had no idea why it was failing.  Where can I find the codestyle rules that will make it pass?

Reply:
I think they are in the setup.cfg file

Reply:
@DanRyanIrish You can probably use https://github.com/asottile/seed-isort-config to help you to populate the `known_third_part` list to pass the `isort` test locally. Then commit all the changes first in the local environment before pushing the changes to the repo. You don't want `isort` to make changes "remotely" for you.

Reply:
I used isort locally. Isort is not run by the CI.